bazel/repository_locations.bzl

```diff
@@ -1315,12 +1315,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (Rust SDK)",
         project_desc = "WebAssembly for Proxies (Rust SDK)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-rust-sdk",
-        version = "0.2.0",
-        sha256 = "010249eac821641b32813670d0ec8a372899a70a7769de5e52e0292d8cd7e0d5",
+        version = "0.2.1",
```

Comment:
Oh, I didn't know that you meant to bundle it the same PR, but I guess that works as well.